main.go

```diff
@@ -61,6 +61,13 @@ func main() {
 	}
 	v = append(v, fmt.Sprintf("spec: %s", specs.Version))
 	app.Version = strings.Join(v, "\n")
+
+	root := "/run/runc"
+	if os.Geteuid() != 0 {
+		runtimeDir := os.Getenv("XDG_RUNTIME_DIR")
+		root = runtimeDir + "/runc"
+	}
```

Comment:
This doesn't handle the case where `XDG_RUNTIME_DIR` is unspecified safely. If it's not set, just use `/run/runc` as usual.